internal/Get-DbaADObject.ps1

```diff
@@ -24,108 +29,65 @@ This program is distributed in the hope that it will be useful, but WITHOUT ANY
 You should have received a copy of the GNU General Public License along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 .EXAMPLE
-Get-DbaADObject -ADObject "contoso\(&(objectCategory=User)(sAMAccountName=ctrlb))"
+Get-DbaADObject -ADObject "contoso\ctrlb" -Type User
```

Comment:
Maybe an example with the silent parameter might be helpful, if it's intended for internal usage